__tests__/__renderer__/classes/FlexibleDayCalendar.js

```diff
@@ -18,6 +18,13 @@ window.$.fn.extend({
     }
 });
 
+jest.mock('../../../renderer/i18n-translator-node-copy.js', () => ({
+    translatePage: jest.fn().mockReturnThis(),
+    getTranslationInLanguageData: jest.fn().mockReturnThis()
+}));
+
+const languageData = {'language': 'en', 'data': {'bla': 'ble'}};
```

Comment:
I know it's not used, but I think it's worth give it meaningful variables.

```suggestion
const languageData = {'language': 'en', 'data': {'dummy_string': 'dummy_string_translated'}};
```